### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by danielpassos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,43 +133,39 @@ In your application info.plist, add the following properties:
...
-## AeroGear UnifiedPush Server
-
-For more information checkout our [tutorial](http://aerogear.org/docs/unifiedpush/aerogear-push-ios/).
 
 ## Documentation
 
-For more details about the current release, please consult [our documentation](http://aerogear.org/ios/).
+For more details about that please consult [our documentation](http://aerogear.org/ios/).
+
+## Demo apps
```
Comment (jcesarmobile) [inline README.md]: Demo apps are Swift, I think we should remove them as this is an Objective-C framework

File: README.md [hunk 2]
```diff
@@ -96,43 +133,39 @@ In your application info.plist, add the following properties:
 
 > NOTE: If your UPS server installation uses a `self-signed certificate`, you can find a quick solution on how to enable support on our [troubleshooting page](https://aerogear.org/docs/unifiedpush/aerogear-push-ios/troubleshooting/#_question_failure_to_connect_when_server_uses_a_self_signed_certificate), as well as links for further information on how to properly enable it on your iOS production applications.
```
Comment (jcesarmobile) [inline README.md]: Link is broken, it should be 

https://aerogear.org/docs/unifiedpush/aerogear-push-ios/guides/#_question_failure_to_connect_when_server_uses_a_self_signed_certificate

**Review by jcesarmobile — CHANGES_REQUESTED** [review]
Just 2 changes

**Comment by danielpassos** [comment]
Done!

**Review by jcesarmobile — APPROVED** [review]

**Pull request merged**